modules/api/sync/package/index.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import psycopg2
+import csv
+import decree_provider
+import decree_repository
+import restriction_provider
+import restriction_repository
+import zone_repository
+
+def lambda_handler(_event, _context):
+    connection = psycopg2.connect(database="ecowater", user="ecowater", password="ecowater", host="pg", port="5432")
+    connection.autocommit = True
+    cursor = connection.cursor()
+
```

Comment:
to do gérer le nettoyage de la base de donnée avant import (sans interuption de service si possible)